### PR TITLE
427 Added DB Transactions to Sequelize Migrations

### PR DESCRIPTION
**Pull request by duselmann**
See issue for details. #427

### CODE REVIEW

**Comment by duselmann** [comment]
While working on the enhancements discussed in slack, I modularized the transactions method. I plan to land the new methods in here and submit for a new PR. However, I felt this change was appropriate for this PR since it is open.

**Comment by duselmann** [comment]
I should note that I only tested "change" actions. We currently have no other actions but I should test them all the same.

File: server/dba/migrations/modules/transaction.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+let Promise = require('bluebird');
+
+let doTransaction = (tableName, queryInterface, operations) => {
+  return queryInterface.sequelize.transaction( (trx) => {
+    return Promise.each(operations, (operation) => {
+      if (operation.operation === 'add') {
```
Comment (duselmann) [inline server/dba/migrations/modules/transaction.js]: a switch statement - sure

Comment (duselmann) [inline server/dba/migrations/modules/transaction.js]: what should we use for the default case? an error?

Comment (dubharmonic) [inline server/dba/migrations/modules/transaction.js]: Sure

Comment (duselmann) [inline server/dba/migrations/modules/transaction.js]: done in following commit

**Comment by duselmann** [comment]
Once this is merged I have a prototype for pre-migrations to submit for review.

**Comment by duselmann** [comment]
I have cleaned up some of the lint issues in my workspace but I have do not want to risk further work contamination again. I will commit it with the next PR if that is okay. (Not sure why it requires more indent on the switch but I do not care if that is what is desired.)

**Pull request merged**